wiki/Tournaments/RMoT_Invitational/1/en.md

```diff
@@ -0,0 +1,115 @@
+# Rapid Monthly osu! Tournament 1
+
+The Rapid Monthly osu! Tournament Invitational \#1 is a tournament organized and ran by the Rapid Tournaments team.
+It is an exclusive tournament where only players who reached semi-finals in one of the last 4 RMoT's get invited.
+
+The tournament brackets can be found [on Challonge.](https://challonge.com/rmotinvitational1). Most of the information here complies with the [forum post](https://osu.ppy.sh/forum/t/538852).
```

Comment:
`The tournament brackets can be found [on Challonge](https://challonge.com/rmotinvitational1). Most of the information here complies with the [forum post](https://osu.ppy.sh/forum/t/538852).`

or

`The tournament brackets can be found [on Challonge.com](https://challonge.com/rmotinvitational1). Most of the information here complies with the [forum post](https://osu.ppy.sh/forum/t/538852).`

I see two fullstops.